internal/server/audit/audit.go

```diff
@@ -31,14 +32,14 @@ type Type string
 type Action string
 
 const (
-	Constraint   Type = "constraint"
-	Distribution Type = "distribution"
-	Flag         Type = "flag"
-	Namespace    Type = "namespace"
-	Rule         Type = "rule"
-	Segment      Type = "segment"
-	Token        Type = "token"
-	Variant      Type = "variant"
+	ConstraintType   Type = "constraint"
+	DistributionType Type = "distribution"
+	FlagType         Type = "flag"
+	NamespaceType    Type = "namespace"
+	RuleType         Type = "rule"
+	SegmentType      Type = "segment"
+	TokenType        Type = "token"
+	VariantType      Type = "variant"
```

Comment:
I created the actual entities for the `audit` package. I was at a crossroads when trying to come up with a name for these types. Let me know what y'all think?

Reply:
I dig it